functions.py

```diff
@@ -11,106 +11,118 @@
 import matplotlib.pyplot as plt
 
 try:
-  from google.colab import files
-  IN_COLAB = True
+    from google.colab import files
+
+    IN_COLAB = True
 except:
-  IN_COLAB = False
+    IN_COLAB = False
 
 MODEL_PATH = Path("model")
 MODEL_FILENAME = "trained_model.pkl"
 DATA_PATH_SERENGETI = Path("images_serengeti")
 DATA_PATH_FUN_EXAMPLES = Path("images_fun_examples")
 
+
 def get_test_images_from_folder(data_path):
-    test_img_list = [str(Path(data_path)/file) for file in os.listdir(data_path) \
-                     if os.path.isfile(Path(data_path)/file) and imghdr.what(Path(data_path)/file) in ["jpeg", "png"]]
+    test_img_list = [
+        str(Path(data_path) / file)
+        for file in os.listdir(data_path)
+        if os.path.isfile(Path(data_path) / file)
+        and imghdr.what(Path(data_path) / file) in ["jpeg", "png"]
```

Comment:
i know this wasn't the main scope of this issue, but once we add this code it'll be hard to change it.

1.  `Path(data_path)` is repeated, if you extract it the conditional will fit in a single line. Additional nice thing is that once data_path is a `Path`, you'll be able to use `data_path.iterdir()` instead of `os.listdir`
2. please extract allowed file extensions to a constant